### DISCUSSION TITLE
why is there python and C# in the codebase

### DISCUSSION THREAD

**Discussion by nicosql** [trailbaseio/trailbase#12] [Q&A]
why is there python and C# in the codebase

**Comment by ignatz** [comment]
If you click on the individual language labels of the GitHub language break down you'll get straight to code search, e.g. https://github.com/search?q=repo%3Atrailbaseio%2Ftrailbase++language%3AC%23&type=code .

In short, these are client libraries. Based on your other question you seem to be familiar with PocketBase. This is equivalent to:
 * https://github.com/vaphes/pocketbase
 * https://github.com/Naquino14/PocketbaseNET

Is there any concern?
